src/pveclib/vec_int512_ppc.h

```diff
@@ -2297,6 +1982,33 @@ vec_mul256x256 (__VEC_U_256 m1, __VEC_U_256 m2);
 extern __VEC_U_640
 vec_mul512x128 (__VEC_U_512 m1, vui128_t m2);
 
+/** \brief Vector 512x128-bit Multiply-Add Unsigned Integer.
+ *
+ *  Compute the 640 bit sum of the product of the 512 bit value m1
+ *  and 128-bit value m2 plus the 512-bit value a2.
+ *  The sum is returned as single 640-bit integer in a
+ *  homogeneous aggregate structure.
+ *
+ *  \note The advantage of this form is that the final 640 bit sum can
+ *  not overflow and carries between stages are eliminated.
+ *  Also applying the addend early (1st multiply stage) reduces the
+ *  live ranges for registers passing partial products for larger
+ *  multiple precision multiplies.
+ *
+ *  |processor|Latency|Throughput|
+ *  |--------:|:-----:|:---------|
+ *  |power8   |224-232| 1/cycle  |
+ *  |power9   |132-135| 1/cycle  |
+ *
+ *  @param m1 vector representation of a unsigned 512-bit integer.
+ *  @param m2 vector representation of a unsigned 128-bit integer.
+ *  @param a2 vector representation of a unsigned 512-bit integer.
+ *  @return homogeneous aggregate representation of the unsigned
+ *  640-bit sum of (m1 * m2) + a2.
+ */
+extern __VEC_U_640
+vec_madd512x128a512 (__VEC_U_512 m1, vui128_t m2, __VEC_U_512 a2);
+
```

Comment:
This seems like a change unrelated to "more doc updates".  NBD, though.

Reply:
I think I missed filling in the description and extern for vec_madd512x128a512() in one of the updates. It was added to vec_int512_runtime earlier and the previous pull included the required the vec_runtime_DYN updates. 

It gets messy in my head :-/